concepts/conditionals-switch/about.md

```diff
@@ -0,0 +1,61 @@
+# About
+
+## General Syntax
+
+Besides the if-statement, JavaScript also has a switch-statement to conditionally execute logic.
+It is used when a single variable needs to be compared to multiple variants.
+The comparison is done by checking for strict equality (`===`), see [concept comparison][concept-comparison].
+For some variable `x`, the switch statement in JavaScript has the following syntax.
+
+<!-- prettier-ignore-start -->
+```javascript
+switch (x) {
+  case option1:
```

Comment:
@SleeplessByte How strongly do you feel about this? Of course this is theoretically correct but I almost never see this in any real code, documentation or tutorial. So even if we explain it a bit in about.md it might be quite confusing to a student that what he/she is introduced to in the introduction.md file does not match the syntax in any other resources.

Would you also be ok if I keep the current syntax as "general" and then add a section in about.md for the block syntax? (This is also how its done on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#block-scope_variables_within_switch_statements).)

Reply:
I don't feel strongly about it. I have _however_ had to fix quite a few production issues because people didn't *know* about them. So not seeing it in the wild is likely more ignorance than it is intentional.

> Would you also be ok if I keep the current syntax as "general" and then add a section in about.md for the block syntax? (This is also how its done on MDN.)

Yep! I like that solution.

Reply:
Could you give me an example of the coding bug/pitfall you saw and fixed? I am not sure I fully understand what you have in mind.

Reply:
Exactly the syntaxerror as mentioned [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#block-scope_variables_within_switch_statements). 

You only see the SyntaxError if the file is loaded. In this case, the developer added a new case statement, without a block, copied the previous one, and expected it to work.

Narrator: it didn't.

Reply:
I understand. Here is the new section: https://github.com/exercism/javascript/pull/1073/commits/7c34deddc077c5c6be5836b784731a8b296dfce6